users/userservice/user-service.js

```diff
@@ -56,7 +56,7 @@ function validatePassword(password) {
 
   const strongPasswordRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[^A-Za-z0-9]).{8,}$/;
 
-  if (!strongPasswordRegex.test(passwordStr)) {
+  if (password==="" || !strongPasswordRegex.test(passwordStr)) {
```

Comment:
The same as before, this validation is already covered by the `regex` test.
```suggestion
  if (!strongPasswordRegex.test(passwordStr)) {
```